apps/frontend/src/app/Components/GeneralAutocomplete.tsx

```diff
@@ -13,17 +13,18 @@ type GeneralAutocompletePropsBase<T extends string> = {
   toExItemLabel?: (v: T) => Displayable | undefined,
   toExLabel?: (v: T) => Displayable | undefined,
   chipProps?: Partial<ChipProps>
+  flattenCorners?: boolean
 }
 export type GeneralAutocompleteProps<T extends string> = GeneralAutocompletePropsBase<T> & { valueKey: T | null, onChange: (v: T | null) => void, } &
   Omit<AutocompleteProps<GeneralAutocompleteOption<T>, false, false, false>, "renderInput" | "isOptionEqualToValue" | "renderOption" | "onChange" | "value">
-export function GeneralAutocomplete<T extends string>({ options, valueKey: key, label, onChange, toImg, toExItemLabel, toExLabel, ...acProps }: GeneralAutocompleteProps<T>) {
+export function GeneralAutocomplete<T extends string>({ options, valueKey: key, label, onChange, toImg, toExItemLabel, toExLabel, flattenCorners = false, ...acProps }: GeneralAutocompleteProps<T>) {

```

Comment:
I would prefer having a `inputProps` parameter here, and then you can apply style to it directly. but applying `inputProps.sx` might overwrite some styling as well without layered merging